pkg/config/config.go

```diff
@@ -77,6 +77,9 @@ type Config struct {
 	Metrics     PrometheusConfig
 	ScaleToZero ScaleToZeroConfig
 
+	Labels      map[string]string
+	Annotations map[string]string
```

Comment:
I think these should have slightly more specific names, maybe
`runtimePodLabels`, `runtimePodAnnotations`. Otherwise it's not clear what these really refer to.